src/zorak/utilities/cog_helpers/_embeds.py

```diff
@@ -449,9 +422,19 @@ def embed_leaderboard(people_list, server_name, server_logo):
     )
     for place, person in enumerate(people_list):
         embed.add_field(
-            name=f"#{place + 1}  -  {person[0].display_name}"
+            name=f"#{place+1}  -  {person[0].display_name}"
             , value=f"Points: {person[1]}"
             , inline=False
         )
 
     return embed
+
+
+def embed_suggestion():
```

Comment:
What's this empty boi doing down here?

Reply:
Not all parts of the file here match with the one I pushed, unsure as to why. I'll look into it when I have a spare moment.

Reply:
Yeah, seems like there are some odd things between this one and the Dev branch. Dunno.